holepunch/__init__.py

```diff
@@ -294,11 +308,11 @@ def holepunch(args):
         print('No changes to make.')
         return True
 
-    print('Changes to be made to: {group_name} [{group_id}]'
-          '\n{hr}\n{perms}\n{hr}'.format(
-              hr='='*60, group_name=group['GroupName'],
-              group_id=group['GroupId'],
-              perms=json.dumps(new_perms, indent=4)))
+    log('Changes to be made to: {group_name} [{group_id}]'
+        '\n{hr}\n{perms}\n{hr}'.format(
+        hr='='*60, group_name=group['GroupName'],
+        group_id=group['GroupId'],
+        perms=json.dumps(new_perms, indent=4)))
```

Comment:
Here I would argue that we would want to print this if `args['--yes'] == False`, even when using the quiet mode, as it is part of the confirmation